web/hooks/use-group.ts

```diff
@@ -205,3 +207,18 @@ export function useGroupContractIds(groupId: string) {
 export function useGroups(groupIds: string[]) {
   return useStoreItems(groupIds, listenForGroup, { loadOnce: true })
 }
+
+export function useIsGroupMember(groupSlug: string) {
+  const [isMember, setIsMember] = useState<any>(undefined)
```

Comment:
Wouldn't this be `boolean | undefined` instead of `any`?

Reply:
yeah I was getting an error because it doesn't recognize the return type of `getUserIsGroupMember` as a boolean